pkg/wildcard/wildcard_test.go

```diff
@@ -54,34 +54,34 @@ func TestMatches(t *testing.T) {
 			matches:   false,
 		},
 		{
-			name:      "wildcard suffix and prefix match",
+			name:      "wildcard suffix and prefix, match",
 			w:         Wildcard("*-kube-*"),
 			candidate: "test-kube-test",
 			matches:   true,
 		},
+		{
+			name:      "wildcard suffix and prefix, no match",
+			w:         Wildcard("*-kube-*"),
+			candidate: "my-kub-controller",
+			matches:   false,
+		},
 		{
 			name:      "no wildcard, only hypens at suffix and prefix",
 			w:         Wildcard("-kube-"),
 			candidate: "test-kube-test",
 			matches:   false,
 		},
 		{
-			name:      "wild card at suffix and prefix, multiple hyphens",
```

Comment:
Is there a reason tests were removed? Pre-existing tests should still pass with this change?